doc/man/man3/crypto_lock_init.3monocypher

```diff
@@ -235,14 +235,14 @@ XChacha20 derives from Chacha20 the same way XSalsa20 derives from
 Salsa20, and benefits from the same security reduction (proven secure
 as long as Chacha20 itself is secure).
 .Sh SECURITY CONSIDERATIONS
-Using this incremental for decryption means authentication is
-incremental as well.
-This allows two errors: forgetting to call
-.Fn crypto_unlock_final
-altogether, and start processing the message before the authentication
-is complete.
+Messages must be completely decrypted before they can be verified.
```

Comment:
Actually, no. Monocypher uses an "encrypt-then-auth" scheme, so you could totally verify before you decrypt. Still, you are right that a reformulation is called for. I vote for something like you wrote for crypto_check:

    Messages are not verified until the call to
    .Fn crypto_unlock_final .
    Make sure to call it and check its return value
    .Em before
    processing the message.

More or less.